bin/pakiti-client

```diff
@@ -452,6 +472,8 @@ sub init () {
         "output"   => "|o=s",
         "rndsleep" => "|r=i",
         "site"     => "=s",
+        "svmonreport" => "|s",
+        "svmon" => "=s",
```

Comment:
a bit nitpicking, but please align the => chars with those below (or above the longer svmonreport option)